tiup: add three docs (#5061)
pingcap/docs: tiup/tiup-component-cluster-audit.md

---
title: tiup cluster audit
---

# tiup cluster audit

The `tiup cluster audit` command is used to view commands executed on all clusters in the history and the execution log of each command.

## Syntax

```sh
tiup cluster audit [audit-id] [flags]
```

> - If you don’t fill in the `[audit-id]`, the table of operation records will be output in reverse chronological order. The first column is the `audit-id`.

Reviewer: ```suggestion
- If you do not fill in the `[audit-id]`, the table of operation records is output in reverse chronological order. The first column is the `audit-id`.
```

- If you fill in the `[audit-id]`, it means checking the execution log of the specified `audit-id`.

## Option

### -h, --help

- Prints the help information.
- Data type: `Boolean`
- Default: false

## Outputs

> - If `[audit-id]` is specified, the corresponding execution log will be output.

Reviewer: ```suggestion
- If `[audit-id]` is specified, the corresponding execution log is output.
```
Please use the present tense in this PR.
